docs/btc-transactions-validity.md

```diff
@@ -1,23 +1,179 @@
 ## Introduction
 
-##  Prerequisites
+## Prerequisites
 
 ## Taproot outputs
 
-## Types of transactions
+## Building Transactions
 
-## Staking
+There are three special transaction types
 
-### Sending a TX
+- Staking Transaction
+- Unbonding Transaction
+- Slashing Transaction
 
-## Unbonding
+### Staking Transaction
 
-### Sending a TX
+A staking transaction is different to the other transactions. It creates a UTXO
+and can only be spent by specific paths. The paths are a predefined spending
+rules enforced by a one of the spending scripts [here] (#spending-scripts).
 
-## Spending taproot outputs
+Requirements:
 
-## Slashing
+1. **Inputs**: References previous UTXOs, provides BTC
+2. **Taproot Output (`staking_output`)**: The vault that locks BTC. Commits
+    to a script tree composed of the three scripts (timelock, unbondng and slashing.)
+    - **Taproot Public Key**: Created in the Taproot output, commits to `m`
```

Comment:
Imo it would be clearer if this whole info:
```
Taproot Public Key: Created in the Taproot output, commits to m (Merkle root)
Merkle Root: Inside Q, commits to spending rules (timelock, unbonding, slashing)
Internal Public Key: Used to derive Q
```
was presented in separate sub-section similiarty to what is done in current impl doc:
```
Taproot outputs are outputs whose locking script is an elliptic curve point Q created as follows:

Q = P + hash(P||m)G
where:

P is the internal public key
m is the root of a Merkle tree whose leaves consist of a version number and a script
For Bitcoin Staking transactions, the internal public key is chosen as:

P = lift_x(0x50929b74c1a04954b78b4b6035e97a5e078a5a0f28ec96d547bfee9ace803ac0)
```